SDL/Event.cc

```diff
@@ -1,7 +1,8 @@
 #include "Event.h"
 
 SDL::modules* SDL::modulesInGPU = new SDL::modules();
-SDL::modulesBuffer<Acc>* SDL::modulesBuffers = new SDL::modulesBuffer<Acc>(devAcc);
+SDL::modulesBuffer<SDL::Dev>* SDL::modulesBuffers = new SDL::modulesBuffer<SDL::Dev>(devAcc);
```

Comment:
I see you changed Acc to SDL::Dev here, but you should do this for all of our objects to be consistent I think. I don't know if this matters since the performance plots look unchanged and this uses SDL::Dev while all the other Buffers still have Acc, but good to be consistent.

Same in all of the header files that define a buffer, to remove confusion we should move the "Acc" named templated parameter to Dev so it's clear what we pass in. See the random examples below.

https://github.com/SegmentLinking/TrackLooper/blob/25e08e79a0def04b7b51e8ec84275e8e84fd8829/SDL/Event.cc#L319

https://github.com/SegmentLinking/TrackLooper/blob/25e08e79a0def04b7b51e8ec84275e8e84fd8829/SDL/MiniDoublet.h#L96-L99

Reply:
> I don't know if this matters since the performance plots look unchanged and this uses SDL::Dev while all the other Buffers still have Acc, but good to be consistent.

right, Acc or Dev does not seem to matter for the buffers at the final implementation level (after all the template parsing is done); however code exposed to CMSSW needs to be templated from a Dev. 

Dev seems to be more appropriate based on Alpaka docs for a buffer https://alpaka.readthedocs.io/en/latest/_images/structure_assoc.png